common/types/constants.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+const (
+	STALE_ENTRY_TIME int64 = 1440 // 1440 blocks (equivalent to 24 hours when block_time = 1min)
+)
+
+// References action enum
+const (
```

Comment:
create a type for this enum 
example:
type MessageType int32

const (
    TEXT   MessageType = 0
    BINARY MessageType = 1
)

Reply:
done